src/components/DeckMechanisms/index.js

```diff
@@ -422,8 +449,8 @@ export default class DeckMechanisms extends React.Component {
     }
 
     if (this.state.turn === 1) {
-      const unplayableSpells = ['W1', 'S10', 'N15']
-      
+      const unplayableSpells = ['W1', 'S10', 'N15', 'N63']
```

Comment:
Let’s remove that change from this pull-request and do it separately with a test? :)